plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/gather_nd.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/gather_nd.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> dPrecisions = {
+        InferenceEngine::Precision::FP32,
+	/*
+        InferenceEngine::Precision::FP16,
```

Comment:
Are all the tests commented fail? I think we should try to support FP16.

Reply:
Uncommneted all the tests and they all passed.

Reply:
Great!